tests/pipeline/test_column.py

```diff
@@ -81,3 +84,41 @@ def test_latest(self):
 
             expected_col_result = self.expected_latest(column, cal_slice)
             assert_frame_equal(col_result, expected_col_result)
+
+    @parameterized.expand([
+        (operator.gt,),
+        (operator.ge,),
+        (operator.lt,),
+        (operator.le,),
+    ])
+    def test_comparison_errors(self, op):
+        for column in TDS.columns:
+            with self.assertRaises(TypeError):
+                op(column, 1000)
+            with self.assertRaises(TypeError):
+                op(1000, column)
+            with self.assertRaises(TypeError):
+                op(column, 'test')
+            with self.assertRaises(TypeError):
+                op('test', column)
+
+    def test_comparison_error_message(self):
+        column = USEquityPricing.volume
+        err_msg = (
+            "'<' not supported between instance of"
+            " 'int' and 'EquityPricing<US>.volume'."
+            " Did you mean to use 'EquityPricing<US>.volume.latest'?"
+        )
+
+        with self.assertRaises(TypeError) as e:
+            column < 1000
+        self.assertEqual(str(e.exception), err_msg)
+
+        with self.assertRaises(TypeError) as e:
+            1000 > column
```

Comment:
Hmm... I was hoping we'd always be able to display the correct operator symbol in the error message; but I guess we can't actually know which one it is without inspecting the stack, since Python just uses `__lt__` instead of `__rgt__`. (It seems reasonable that the interpreter would make use of its own stack to get that information when generating the built-in errors in Python 3.7+, though.)

Since the `<` symbol in this error message is not the same as the `>` symbol in the code that caused it, I think it's probably better to not display a symbol at all, rather than one that's only correct in half the cases. On the plus side, it should simplify the code a bit. I'll go ahead and make the change; let me know if you have other ideas, though.